src/node.rs

```diff
@@ -75,7 +82,7 @@ impl<'a> CustomWidget for Node<'a> {
 
     type State = NodeState;
 
-    fn set<C>(self, ui_id: UiId, ui: &mut Ui<C>) {
+    fn update<C>(self, state: NodeState, ui_id: UiId, ui: &mut Ui<C, Node<'a>>) -> WidgetUpdate<NodeState> {
```

Comment:
You might have to add `where C: CharacterCache` here

CharacterCache is not yet exposed (it's in `graphics::character::CharacterCache`, but we should expose it from the conrod lib too). I'll do it now.